src/EasyCaching.LiteDB/EasyCaching.LiteDB.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../build/version.props" />
+  <Import Project="../../build/releasenotes.props" />
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Owners>maikebing</Owners>
+    <Authors>maikebing</Authors>
+    <Version>1.0.1</Version>
```

Comment:
Can we read this version with `$(EasyCachingXXXPackageVersion)`  from `version.props`?

Reply:
Okay, I've pushed the fix.